src/sync.d

```diff
@@ -933,10 +933,16 @@ final class SyncEngine
 		if(encodeComponent(path).length < maxPathLength){
 			// path is less than maxPathLength
 
-			// skip unexisting symbolic links
-			if (isSymlink(path) && !exists(readLink(path))) {
-				log.vlog("Skipping item - symbolic link: ", path);
-				return;
+			if (isSymlink(path)) {
+				// if config says so we skip all symlinked items
+				if (cfg.getValue("skip_symlinks") == "true") {
+					return;
```

Comment:
If you can add in a log.vlog statement here as to why the symbolic link was skipped this should good to go and merged. Tested locally functionality wise all OK.

Reply:
coolio, I'll do that :)